en/components/dock-manager.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Dock Manager | Layout Controls | Ignite UI for Angular | Infragistics
+_description: 
+_keywords: dock manager, layout, igniteui for angular, infragistics
+---
+## Dock Manager
+
+The Web Components Dock Manager is a component that uses panes to manage the application’s layout and allows the end user to customize the UI by unpinning, resizing, moving and hiding panes.
+
+### Demo
+
+<div class="sample-container loading" style="height: 600px">
+    <iframe id="dock-manager-overview-iframe" src='{environment:dvDemosBaseUrl}/layouts/dock-manager-sample' width="100%" height="100%" seamless frameBorder="0" onload="onXPlatSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn" data-iframe-id="dock-manager-overview-iframe" data-demos-base-url="{environment:dvDemosBaseUrl}">View on StackBlitz
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+
+### Setting Up
+The DockManager is available for download as an npm package through ProGet feed.
```

Comment:
Remove this sentence. Instructions for installing ProGet package will be placed somewhere else.

Reply:
Add a statement like this:

"The Dock Manager is a standard [web component](https://developer.mozilla.org/en-US/docs/Web/Web_Components) and as such can be used in an Angular application."